src/edit_session.js

```diff
@@ -187,6 +190,35 @@ class EditSession {
         return this.doc;
     }
 
+    /**
+     * Get "widgetManager" from EditSession
+     * 
+     * @returns {LineWidgets} object
+     */
+    get widgetManager() {
+        const widgetManager = new LineWidgets(this);
```

Comment:
how does this work when both consumers want to use `editor.widgetManager` and we want to use it internally? Doesn't this mean that when a consumer calls the widgetManager all the existing widgets currently in the editor get deleted?

Reply:
nice catch. Will be adding the check to only create widgetManager if it doesn't exists.